assets/src/edit-story/app/media/utils/useUploadVideoFrame.js

```diff
@@ -56,15 +60,26 @@ function useUploadVideoFrame({ updateMediaElement }) {
         featured_media: posterId,
         post: storyId,
       });
+      await preloadImage(poster);
```

Comment:
Now we are loading the full image in the browser, force preload of image to stop jumping around.

Reply:
Done.